plugins/json_processor/src/json_processor/component.py

```
@@ -0,0 +1,228 @@
+            except Exception as e:
+                logger.warning(f"Failed to apply fraction sampling: {e}")
+
+        return data
+
+    @classmethod
+    def process_info(cls) -> Info:
+        return Info(
+            name="JSONTransform",
+            version="1.0.0",
+            description="Мощный компонент трансформации\
+             данных с поддержкой JSON операций",
```

Ruff (0.12.2)

String contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF001)